docs/source/ModelingApproach.rst

```diff
@@ -305,10 +305,10 @@ battery model.
 Solar PV
 --------
 
-Solar photovoltaics (PV) is modeled using PySAM, a python wrapper for
-the System Advisory Model (SAM). Standard values are used for the PV
-model, although the user can select the PV system capacity, the tilt
-angle, and the orientation.
+Solar photovoltaics (PV) is modeled using PySAM, a python wrapper for the
+System Advisory Model (SAM), using the PVWatts module. Standard values are
```

Comment:
Pretty minor wording, but... can we change "Standard" to "SAM defaults"? "Standard" is very open to interpretation, saying it's based on what the SAM team thinks is a reasonable default is more defensible.